CMakeLists.txt

```diff
@@ -106,6 +106,8 @@ if(WITH_XC_ALL)
     if(UNIX AND NOT APPLE)
         set(WITH_XC_FDOSECRETS ON)
     endif()
+elseif(WITH_XC_BROWSER_PASSKEYS)
+    set(WITH_XC_BROWSER ON)
```

Comment:
This is already taken care of in another PR and is unrelated to yours.

Reply:
Forgot to remove this as well after getting things set up, I'll remove it.